packages/cli/commands/project/open.js

```diff
@@ -0,0 +1,70 @@
+const open = require('open');
+const {
+  addAccountOptions,
+  addConfigOptions,
+  getAccountId,
+  addUseEnvironmentOptions,
+  addTestingOptions,
+} = require('../../lib/commonOpts');
+const { loadAndValidateOptions } = require('../../lib/validation');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const { logger } = require('@hubspot/cli-lib/logger');
+const {
+  getProjectConfig,
+  getProjectDetailUrl,
+  verifyProjectExists,
+} = require('../../lib/projects');
+const { projectNamePrompt } = require('../../lib/prompts/projectNamePrompt');
+
+const i18nKey = 'cli.commands.project.subcommands.open';
+
+exports.command = 'open';
```

Comment:
Can this be updated to `open [--project]` to help make the project option more discoverable?